docs/docs/code/bash/README.md

```diff
@@ -0,0 +1,199 @@
+# Writing Bash in Code Steps
+
+Prefer to write quick scripts in bash? We've got you covered.
```

Comment:
Capitalize "bash" here

Reply:
I saw some other places in this doc where Bash should be capitalized, as well. I'll make sure it's cleaned up in the product, too.